src/libraries/System.Runtime/tests/System/TimeOnlyTests.cs

```diff
@@ -483,12 +483,15 @@ public static void TryFormatTest()
             Assert.False(timeOnly.TryFormat(buffer.Slice(0, 3), out charsWritten));
             Assert.False(timeOnly.TryFormat(buffer.Slice(0, 3), out charsWritten, "r"));
             Assert.False(timeOnly.TryFormat(buffer.Slice(0, 3), out charsWritten, "O"));
-        }
 
-        [Fact]
-        public static void InvalidFormattingWithInterpolatedStringTest()
-        {
-            TimeOnly timeOnly = TimeOnly.FromDateTime(DateTime.Now);
+            Assert.Throws<FormatException>(() => {
+                    Span<char> buff = stackalloc char[100];
+                    timeOnly.TryFormat(buff, out charsWritten, "u");
+                });
+            Assert.Throws<FormatException>(() => {
+                    Span<char> buff = stackalloc char[100];
+                    timeOnly.TryFormat(buff, out charsWritten, "dd-yyyy");
```

Comment:
Am I understanding correctly that there are some format specifiers that have special meaning, others that aren't recognized and are just treated as custom specifiers, and others that are considered erroneous and are failures?  Do we have sufficient test coverage for each of those categories?

Reply:
We have enough tests for formatting in general through testing with `ToString()` which cover all cases you mentioned. If you are talking specifically about TryFormat, we test with the `o` and `r` formats only. I was focusing here to add tests for the case I am fixing (as I am thinking of porting this fix to 6.0). but I can try adding such tests either now or in another PR.

Reply:
> We have enough tests for formatting in general through testing with ToString() which cover all cases you mentioned

Ok, thanks.